frontend/src/scenes/billing/v2/test/Billing.tsx

```diff
@@ -60,8 +72,7 @@ export function BillingV2({ redirectPath = '' }: BillingV2Props): JSX.Element {
         </Link>
     )
 
-    const products =
-        enterprisePackage && billing?.products_enterprise ? billing?.products_enterprise : billing?.products
+    const products = billing?.products
```

Comment:
Show directly the products, as these would either match the standard products (if not subscribed), or the plan products (if subscribed)